### PR TITLE
install test case coverage check package

### PR DESCRIPTION
**Pull request by skqksh**
## Proposed changes

- install test case coverage check package

## Types of changes

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- N/A

## Further comments

- to update dependencies and refactoring codes, it needs lots of test cases sufficiently
- test coverage checker will be helpful to make test cases
- try with commands following : npm run cover

### CODE REVIEW

**Comment by kjhman21** [comment]
@jimni1222 Please take a look.

File: package.json [hunk 1]
```diff
@@ -25,7 +25,11 @@
     "intFDTest": "mocha --grep INT-FD/ test/intTest.js",
     "intFDRTest": "mocha --grep INT-FDR/ test/intTest.js",
     "intSIGVALITest": "mocha --grep INT-SIGVALI/ test/intTest.js",
-    "intAPITest": "mocha --grep INT-API/ test/intTest.js"
+    "intAPITest": "mocha --grep INT-API/ test/intTest.js",
+    "cover": "npm run build && nyc --silent mocha test/packages/caver.utils.js && nyc --silent --no-clean mocha test/packages/caver.klay.net.js && npm run cover:scenario && npm run cover:wallet && npm run cover:report",
+    "cover:scenario": "nyc --silent --no-clean npm run serTest",
```
Comment (jimni1222) [inline package.json]:
```suggestion
    "cover:serialization": "nyc --silent --no-clean npm run serTest",
```
npm run serTest tests serialization :) So i think cover:serialization is better.

File: package.json [hunk 2]
```diff
@@ -25,7 +25,11 @@
     "intFDTest": "mocha --grep INT-FD/ test/intTest.js",
     "intFDRTest": "mocha --grep INT-FDR/ test/intTest.js",
     "intSIGVALITest": "mocha --grep INT-SIGVALI/ test/intTest.js",
-    "intAPITest": "mocha --grep INT-API/ test/intTest.js"
+    "intAPITest": "mocha --grep INT-API/ test/intTest.js",
+    "cover": "npm run build && nyc --silent mocha test/packages/caver.utils.js && nyc --silent --no-clean mocha test/packages/caver.klay.net.js && npm run cover:scenario && npm run cover:wallet && npm run cover:report",
```
Comment (jimni1222) [inline package.json]:
```suggestion
    "cover": "npm run build && nyc --silent mocha test/packages/caver.utils.js && nyc --silent --no-clean mocha test/packages/caver.klay.net.js && npm run cover:serialization && npm run cover:wallet && npm run cover:report",
```

**Comment by jimni1222** [comment]
When i tried to run 'npm run cover', 'sh: start: command not found' error is occured at cover:report.
What should i have to add for that ?

**Comment by skqksh** [comment]
> When i tried to run 'npm run cover', 'sh: start: command not found' error is occured at cover:report.
> What should i have to add for that ?

Can I know your npm version ( npm -v )
I'm using v6.4.1
I hope you update npm version and retry it ( npm i -g npm )
or If you have to use some version of npm. please tell me

**Comment by jimni1222** [comment]
> > When i tried to run 'npm run cover', 'sh: start: command not found' error is occured at cover:report.
> > What should i have to add for that ?
> 
> Can I know your npm version ( npm -v )
> I'm using v6.4.1
> I hope you update npm version and retry it ( npm i -g npm )
> or If you have to use some version of npm. please tell me

I use npm v6.12.1.

**Comment by skqksh** [comment]
> > > When i tried to run 'npm run cover', 'sh: start: command not found' error is occured at cover:report.
> > > What should i have to add for that ?
> > 
> > 
> > Can I know your npm version ( npm -v )
> > I'm using v6.4.1
> > I hope you update npm version and retry it ( npm i -g npm )
> > or If you have to use some version of npm. please tell me
> 
> I am using npm v6.12.1.

please check following
- npm install
- check commit location (now on eb08801)
- npm run cover

I'm using cmd or powershell with vscode. 

Can you tell me other environment for develop?

**Comment by jimni1222** [comment]
> > > > When i tried to run 'npm run cover', 'sh: start: command not found' error is occured at cover:report.
> > > > What should i have to add for that ?
> > > 
> > > 
> > > Can I know your npm version ( npm -v )
> > > I'm using v6.4.1
> > > I hope you update npm version and retry it ( npm i -g npm )
> > > or If you have to use some version of npm. please tell me
> > 
> > 
> > I am using npm v6.12.1.
> 
> @jimni1222
> please check following
> 
> * npm install
> * check commit location (now on [eb08801](https://github.com/klaytn/caver-js/commit/eb0880182f7c4f61bd503f9d2c2266d81e959cd4))
> * npm run cover
> 
> I'm using cmd or powershell with vscode.
> 
> Can you tell me other environment for develop?

I did npm install, and rebased my test branch on your blue-cover-test branch.
What role does the 'start' command do?

**Comment by skqksh** [comment]
> > > > > When i tried to run 'npm run cover', 'sh: start: command not found' error is occured at cover:report.
> > > > > What should i have to add for that ?
> > > > 
> > > > 
> > > > Can I know your npm version ( npm -v )
> > > > I'm using v6.4.1
> > > > I hope you update npm version and retry it ( npm i -g npm )
> > > > or If you have to use some version of npm. please tell me
> > > 
> > > 
> > > I am using npm v6.12.1.
> > 
> > 
> > @jimni1222
> > please check following
> > 
> > * npm install
> > * check commit location (now on [eb08801](https://github.com/klaytn/caver-js/commit/eb0880182f7c4f61bd503f9d2c2266d81e959cd4))
> > * npm run cover
> > 
> > I'm using cmd or powershell with vscode.
> > Can you tell me other environment for develop?
> 
> I did npm install, and rebased my test branch on your blue-cover-test branch.
> What role does the 'start' command do?

just to open coverage/index.html

**Comment by jimni1222** [comment]
> @jimni1222
> 
> just to open coverage/index.html

ah, okayokay! 
I think for using command 'start' need to set other thing seperately.
I can check the coverage/index.html file the error is occured though, so if that command is only for opening the file, then how about just remove that command?

**Comment by skqksh** [comment]
> > @jimni1222
> > just to open coverage/index.html
> 
> ah, okayokay!
> I think for using command 'start' need to set other thing seperately.
> I can check the coverage/index.html file the error is occured though, so if that command is only for opening the file, then how about just remove that command?

As you wish ~ 
I did additional commit

**Pull request merged**